andela/AndelaEatsUI
# #162628573 Super admin adds other users as admin

#### What does this PR do?
- Super admin adds other users as admin

#### Description of Task to be completed?
- Implement functionality to ensure that a super admin can add other users as admins
- write tests for this

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-super-admin-add-admins-162628573`
- run these commands `npm run build` and `npm run dev` 
- Log in as a user and hover over the caret on the right of your profile picture. If you are an admin, you should see a link to the `Admin Dashboard`, navigate to the `Admin Dashboard` and click on `Users` from the sidebar. You should now see a form where you enter the user's email that you want to become an admin.

#### What are the relevant pivotal tracker stories?
- [#162628573](https://www.pivotaltracker.com/story/show/162628573)

#### Background Context
- N/A
#### Screenshots (If Applicable)
- N/A
